modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Operation.java

```diff
@@ -134,4 +134,47 @@ public Message installApplication(ApplicationWrapper applicationWrapper) throws
             throw new MDMAPIException(msg, e);
         }
     }
+
+	@POST
+	@Path("uninstallApp/{tenantDomain}")
```

Comment:
Is it required to have tenant domain at the request path it self? Also, it's not recommendable to have a verb like "uninstallApp" to name a REST Resource path.